css/Components/FeedbackSlider.scss

```diff
@@ -0,0 +1,43 @@
+@import "../Theme/Color";
+
+$FeedbackSlider-dots-size: 9px;
+
+.FeedbackSlider-dots {
+  width: 100%;
+  margin: 0;
+
+  list-style: none;
+
+  li {
```

Comment:
I think you can use the `class` names the slider has to customize the appearance, avoiding styling directly `html tags`. Or am I overlooking something?